index.js

```diff
@@ -29,13 +29,14 @@ var resUser = module.exports = function (chatId, callback) {
         uri: url + token + '/' + method + '?' + type + '=' + chatId,
         json: true
     }, function (error, response, body) {
-        if (!error && response.statusCode == 200) {
-            var result = body && body.result;
-            if(result)callback(error, result);
-            else callback(new Error('Username not found'));
+        if (error) {
```

Comment:
Nice job. I would prefer to structure the code so that we handle the errors first, `return`ing early where necessary and **finally** handle the success scenario. So that this would have been:

```js
if (error) {
    return callback(error);
}
if (response.statusCode !== 200 || !body.result) {
    error = new Error("Username not found");
    return callback(error);
}
return callback(null, body.result);
```